equinor/EchoComponents
# chore: #62087 Security updates + update storybook to use webpack 5

## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


### Description
We could not upgrade the following packages for months now because they need webpack 5 in storybook to work:
![image](https://user-images.githubusercontent.com/13872908/153370592-a43f7483-a33b-40fa-809a-9faceae00261.png)

Webpack 5 support for storybook is not finalized, need to install separate packages to support it:
- @storybook/builder-webpack5
- @storybook/manager-webpack5
- Added the below part to package json to tell storybook to use webpack 5:
```js
    "resolutions": {
        "@storybook/react/webpack": "^5"
    }
```
- Needed to adjust storybook's webpack config to be able to handle `svg`s

Also updated other updated dependencies:

Before updates
![image](https://user-images.githubusercontent.com/13872908/155484844-0a1e8f8a-63f8-4bbe-948c-5746d0bcc703.png)

After updates
Couldn't updated eds core: it has some breaking changes. Needs coordinated updates throughout Echo repos.
Sortablejs: after upgrade some import errors are thrown in build. Needs further ivestigation.
![image](https://user-images.githubusercontent.com/13872908/155485733-88b2746e-4021-465b-9cfa-b9dc44923f32.png)

![image](https://user-images.githubusercontent.com/13872908/155485957-c0184242-255b-494a-ac13-57235cddbf45.png)

## How to test
- Run storybook: check if everything looks and behaves the same as before (testing webpack 5 + storybook setup)
- Run build: check if it throws any errors (testing rollup)
- Yalc it to echopediaWeb: see if it works as expected (contextMenu's, dropdowns, etc. components from EchoComponents looks and behaves properly)
